FileMapper/find_driveid_api.js

```diff
@@ -0,0 +1,234 @@
+/**
+ * Tester function
+ */
+function testerFunction() {
+  var path = "C:\\Documents\\Muskan\\sheet1.xls";
+  Logger.log(findDriveMapping(path));
+}
+
+/**
+ * Finds the Drive Mapping for the given local 
+ * mapping using the config data
+ *
+ * @param {String} localPath Local Path whose mapping 
+ * is to be found
+ * @return {String} driveMapping The Drive id of the 
+ * mapped file/folder if found, otherwise it contains
+ * "Mapping Not Found, Ask the User".
+ */
+function findDriveMapping(localPath) {
+  var path = localPath;
+
+  // replaced back slash with forward slash
+  // to handle Mac file path
+  path = path.replace(/\//g, "\\");
```

Comment:
since we are looking it up in config are we showing \\ in the config as well?

Reply:
Fixed